### ISSUE TITLE
Use vec2wav for Speech to Speech Voice conversion

### ISSUE THREAD

**Issue by rishikksh20** [cantabile-kwok/CTX-vec2wav#9]
Hi @cantabile-kwok ,

I am curious to know have you tried this model for zero shot voice conversion use case ?
Idea is very simple:

```
Source voice speech -> semantic token -> vec2wav (with target voice prompt) -> Target voice speech
```
We can easily calculate semantic token from pretrained HuBert or VQ-wav2vec etc.

**Comment by cantabile-kwok** [comment]
Yes, this is straightforward, and the result seems decent. When using HuBERT tokens, about 2-3 seconds of prompt speech can get speech output that sounds quite similar.
However, there are still some tricky things to put this method in real-life cases. As far as I know, VQ-wav2vec still carries some speaker information, and HuBERT reconstructs badly for short speech segments.

**Comment by rishikksh20** [comment]
@cantabile-kwok https://arxiv.org/abs/2312.08676 
SEF-VC architecture is same as CTX-vec2wav.
